dags/bqetl_ctxsvc_derived.py

```diff
@@ -95,6 +95,7 @@
         ],
         date_partition_parameter="submission_date",
         depends_on_past=False,
+        parameters=["submission_date:DATE:{{ds}}"],
```

Comment:
not sure if it matters but it looks like `parameters` is supposed to be a tuple.